winpr/libwinpr/sspi/NTLM/ntlm_compute.c

```diff
@@ -248,15 +248,16 @@ BOOL ntlm_write_ntlm_v2_response(wStream* s, const NTLMv2_RESPONSE* response)
 
 void ntlm_current_time(BYTE* timestamp)
 {
-	FILETIME filetime = { 0 };
-	ULARGE_INTEGER time64 = { 0 };
+	union
+	{
+		FILETIME ft;
+		ULARGE_INTEGER ul;
+	} now = { 0 };
 
 	WINPR_ASSERT(timestamp);
 
-	GetSystemTimeAsFileTime(&filetime);
-	time64.u.LowPart = filetime.dwLowDateTime;
-	time64.u.HighPart = filetime.dwHighDateTime;
-	CopyMemory(timestamp, &(time64.QuadPart), 8);
+	GetSystemTimeAsFileTime(&now);
+	CopyMemory(timestamp, &(now.ul.QuadPart), sizeof(now.ul.QuadPart));
```

Comment:
Why are we using this `union` magic here?
Wouldn't a simple `CopyMemory(timestamp, &now, sizeof(now));` work just fine?